<ui version="4.0" >
 <class>ProcWizardThresholdPage</class>
 <widget class="QWidget" name="ProcWizardThresholdPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="4" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Select some authorities from the list and then choose the authority threshold and robustness factor.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QListWidget" name="userListWidget" >
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="addPushButton" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>&amp;Add >></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePushButton" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>&lt;&lt; &amp;Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2" >
    <widget class="QListWidget" name="authListWidget" >
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Select authorities.&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Threshold Options</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="whatsThis" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">The threshold is the number of authorities that must pool together their secret key shares to decrypt the final result.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">You should set this high enough so that you do not put too much power into the hands of small number of authorities, but not so high that it is impractical for this many authorities to be present. &lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="text" >
           <string>&amp;Threshold:</string>
          </property>
          <property name="buddy" >
           <cstring>thresholdSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="thresholdSpinBox" >
          <property name="value" >
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="whatsThis" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">The robustness factor is the fraction of the threshold that is required at each stage.  If in doubt, leave this at the default of 1.&lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="text" >
           <string>Robustness &amp;factor:</string>
          </property>
          <property name="buddy" >
           <cstring>robustnessSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="robustnessSpinBox" >
          <property name="value" >
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>userListWidget</tabstop>
  <tabstop>addPushButton</tabstop>
  <tabstop>removePushButton</tabstop>
  <tabstop>authListWidget</tabstop>
  <tabstop>thresholdSpinBox</tabstop>
  <tabstop>robustnessSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
